### ISSUE TITLE
Text ingredient from IFTTT to Google home ending on 8/31/2022

### ISSUE THREAD

**Issue by DarthNucifera** [OmerTu/GoogleHomeKodi#357]
IFTTT just made an announcement that effective 8/31/2022, all Google assistant Applets will be archived. A new version of the Google Home integration is available, but does not support "Simple phrase" or "text ingredients". Sadly, that effectively kills this project.

**Comment by pinn73** [comment]
@OmerTu - can you confirm this is the case, please?

**Comment by baticag** [comment]
Seen that as well, but hope it will continue working somehow

**Comment by MrAvana** [comment]
> @OmerTu - can you confirm this is the case, please?

https://ifttt.com/explore/google-assistant-changes

**Comment by rainman74** [comment]
Yes, it will end! IFTTT for Google Assistant will then be completely useless!

**Comment by terryo52404** [comment]
I hope this project will continue.  Issue #352 says they have gotten it to work locally with Siri/Homepod. I am willing to buy a Homepod to continue using it, as my disabled husband depends on this while using Kodi.

**Comment by Brad3rs** [comment]
> I hope this project will continue.
> 
> Eta: I didn't realize that the Siri option just called the IFTTT macros. I'm still willing to buy hardware if there are any other options.

another option that works well for us was to temporarily bail on google (not what I wanted to do) but we just got a cheap echo dot and followed this tutorial https://kodi-connect.github.io/  and it was easy to set up and works really well for now.
the commands are similar to this project, you can just say Alexa play bla bla  or whatever and there's been no problems on a pi3 running Libreelec Leia.  a couple of niggles like no command to  play from Youtube but I got around that with this awesome Kodi addon to add any Youtube strm file (channel or playlist) into the library https://github.com/evantaur/plugin.youtube2kodilibrary so we just added what we would have searched for anyway, it all works flawlessly.
shame about the IFTTT google situation, this was an excellent and well managed project but for anyone looking for an alternative, this was a quick and efficient workaround for us.
